book/from_flat_php_to_symfony2.rst

```diff
@@ -705,7 +737,7 @@ for example, the list template written in Twig:
         </ul>
     {% endblock %}
 
-The corresponding ``layout.html.twig`` template is also easier to write:
+The corresponding ``layout.html.php`` template is also easier to write in Twig:
```

Comment:
Actually if you look up closely , in these chapter you first write `layout.html.php` and `list.html.php` and then, where i made a change, you rewrite these to templates with Twig.